main.py

```diff
@@ -33,13 +34,11 @@ async def membercardsranked(ctx, *args):
         return
     clan_tag = args[0]
 
-    # TODO: Why does the bot stop typing so quickly (seems there's a 10 second limit)? Function below hasn't finish completing yet and it finishes typing
     async with ctx.typing():
         # TODO: might want some try catch before sending result in case API requests fail or something...
         print("Started fetch ranked members processing...")
         clan_members_ranked = clan_members_ranker.get_clan_cards_rank(clan_tag)
         print("Completed fetch ranked members processing")
-        await asyncio.sleep(15)
```

Comment:
Yea....I was testing the bot typing animation....and forgot to remove this earlier so this was adding an extra 15 seconds on delay 😢 but not it's ⚡ fast